proposals/collection-literals.md

```diff
@@ -825,6 +825,8 @@ However, given the breadth and consistency brought by the new literal syntax, we
 [unresolved]: #unresolved-questions
 
 Hopefully small questions:
+        
+* Can/should the compiler emit Array.Empty for `[]`.  Should we mandate that it always does this?  Should we make it an option?  Should we disallow and say it's always a fresh instance?  I (cyrus) strongly think it should mandate Array.Empty to avoid allocations whenever possible.
```

Comment:
```suggestion
* Can/should the compiler emit Array.Empty for `[]`?  Should we mandate that it does this, to avoid allocations whenever possible?
```